docs/common/Version.xml

```diff
@@ -40,6 +40,6 @@
     structure that we can use bookinfo/date and other elements for other
     purposes.</para>
 
-    <para>☺</para>
+    <para>☺ © Γ∏</para>
```

Comment:
Not sure what this change is. Could it be a bad character introduced accidentally?